data/wanderers middle.txt

```diff
@@ -1766,7 +1766,51 @@ mission "Wanderers: Sestor Search: Varu K'prai"
+	on offer
+		conversation
+			`As you bring your ship in for landing at an empty pad, you notice the captain of an adjacent Behemoth inspecting his ship, which is raked with projectile punctures.`
+			choice
+				`	(Approach the captain and try to find out what happened)`
+					goto approach
+				`	(Ignore him.)
+					decline
+			label approach
+			`	You get a better view of the Behemoth——the "Heart of Gold", as indicated on the ship's stern——as you walk up to it. The rear of the ship is badly hammered, and black soot-like stains streak forward across the hull. You can't help but notice that at several projectile entry along the hull, the metal is bent smoothly inwards, almost as if the penetrator had seared or melted its way through the ship's armor. The captain of the Behemoth looks up from his inspection. Seeing you, he remarks, "We were lucky to get out of that one alive!"
+			choice
+				`	"How did your ship end up in such a state?"`
+				`	"Did those damn pirates maul this freighter to pieces?"`
+			`	"I'm not quite sure," the captain replies. "We were shot by a fleet of hostile ships that our computer couldn't identify. They didn't look at all like a run-of-the-mill pirate ship, though, and they were kitted out with tech I've never seen before."`
+			choice
+				`	"How many ships were there? What did they look like?"`
+				`	"Where did this fight take place?"`
+				`	"You should be glad you're still in one piece."
+					goto glad
+			`	"Well," the pilot continues, "we had hired a beefy Leviathan escort to Farpoint in the Alnitak system. When we were going in for landing, two of these hostile ships jumped in from the northwest, the direction of all the anarchist colonies around here. The ships weren't big, but they were tough——our four heavy laser turrets weren't enough to produce even a dent in their shields. They immediately shredded the Leviathan warship, and nearly disabled our ship, too. It was lucky I decided that jumping would be faster than landing. We warped out of the system while in atmosphere, saving the ship, but now I'm in debt to all my crew because I couldn't meet my rush delivery deadline."`
+			choice
+				`	"You're lucky as hell to be here today."`
+					goto glad
+				`	"Thanks for the information. I think that these hostile ships are the ones I'm looking for."`
+			`	"No problem, kiddo," the captain says, although he looks awed that you are even attempting to tackle such powerful ships. "Remember, northwest of Farpoint. If they're anarchists of any sort, they'll be hiding in one of those pirate systems up there. Take care of yourself out there; those are some dangerous ships that jumped us." With that, he bids you farewell.`
+					decline
+			label glad
+			`	"Yeah, those ships would have obliterated our Heart of Gold if we hadn't jumped pronto." The pilot looks at you, then asks quizzically, "Say, do you know anything about those aggressors?"`
+			choice
+				`	"Yeah, those were Korath ships that jumped you."`
```

Comment:
I realize it's informal and I'll switch it out, but I don't see the problem with using informal language.

As for your "nowhere else in an 'official' capacity", try the Oxford Dictionary of English. Official enough? Informal, yes, but wholly recognized.
![image](https://cloud.githubusercontent.com/assets/20450076/23679083/a650ea6a-0354-11e7-8dae-c782b337a548.png)